array-loops.js

```diff
@@ -2,34 +2,42 @@
 // Opdracht 1
 // --------------------------
 
-const names = ["Henk", "Piet", "Fred", "Joop"];
+let names = ["Henk", "Piet", "Fred", "Joop"];
```

Comment:
Je hoeft hier geen let van te maken. 

De const slaat in dit geval op het formaat van de array (dus een inhoud van vier Strings) en niet op de inhoud van de strings. Je kunt dus prima de inhoud (de vier strings dus) aanpassen zonder dat je er een let van hoeft te maken. Wat je nu hebt gedaan met het aanmaken van de let variabele is dat het formaat van de array aanpasbaar is.